### ISSUE TITLE
Error uploading

### ISSUE THREAD

**Issue by Lofix18** [YezerSTN/INSSIST#16]
hi , 
thank you so much for helping people like me who cant afford neither ios nor insist , but eventually i installed the latest version you just uploaded and it is giving an error while uploading reels, i tried reinstalling it thrice and tried on 2 diff. accounts still same issue. is there any fix for this issue.. btw you are life saver sir.

**Comment by YezerSTN** [comment]
test
https://github.com/YezerSTN/INSSIST/releases/tag/v23.6.10

**Comment by Lofix18** [comment]
i have tried this one too, actually when i tried the insist it got uploaded, but when i use crack it shows unknown error at end of upload, i don't know why this happening, but i would be glad if you help fixing it :))

**Comment by YezerSTN** [comment]
I think it is because you have two INSSIST at the same time.

 Try removing the ORIGINAL INSSIST and closing your browser. 
Then install the CRACKED INSSIST, because having two versions of INSSIST can cause errors.

**Comment by 2022Sius** [comment]
Yes today, i experience that video cant be uploaded, posted and so on, hope will be better soon, thank s

**Comment by YezerSTN** [comment]
This version has already been tested and I can upload reels.

https://github.com/YezerSTN/INSSIST/releases/tag/v23.7.3
